src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CodeActionEndpoint.cs

```diff
@@ -80,9 +80,9 @@ public async Task<CommandOrCodeActionContainer> Handle(CodeActionParams request,
             var hostDocumentIndex = sourceText.Lines.GetPosition(linePosition);
             var location = new SourceLocation(hostDocumentIndex, (int)request.Range.Start.Line, (int)request.Range.Start.Character);
 
-            var context = new RazorCodeActionContext(request, codeDocument, location);
+            var context = new RazorCodeActionContext(request, document, codeDocument, location);
```

Comment:
Nit; `documentSnapshot` instead of `document` to clarify difference between it and the `codeDocument`. This would also match the [constructor](https://github.com/dotnet/aspnetcore-tooling/pull/2182/files#diff-a8c809a119474c1e7c08d2d3575a608bR13).